arangod/Statistics/ServerStatistics.cpp

```diff
@@ -37,27 +37,51 @@ TransactionStatistics::TransactionStatistics(MetricsFeature& metrics)
                            "Number of transactions committed")),
-      _intermediateCommits(_metrics.counter(
-          "arangodb_intermediate_commits", 0,
-          "Number of intermediate commits performed in transactions")),
+      _intermediateCommits(
+        _metrics.counter("arangodb_intermediate_commits", 0,
+                         "Number of intermediate commits performed in transactions")),
       _exclusiveLockTimeouts(
-          _metrics.counter("arangodb_collection_lock_timeouts_exclusive", 0,
-                           "Number of timeouts when trying to acquire "
-                           "collection exclusive locks")),
-      _writeLockTimeouts(_metrics.counter(
-          "arangodb_collection_lock_timeouts_write", 0,
-          "Number of timeouts when trying to acquire collection write locks")),
-      _lockTimeMicros(_metrics.counter(
-          "arangodb_collection_lock_acquisition_micros", 0,
-          "Total amount of collection lock acquisition time [μs]")),
+        _metrics.counter("arangodb_collection_lock_timeouts_exclusive", 0,
+                         "Number of timeouts when trying to acquire "
+                         "collection exclusive locks")),
+      _writeLockTimeouts(
+        _metrics.counter("arangodb_collection_lock_timeouts_write", 0,
+                         "Number of timeouts when trying to acquire collection write locks")),
+      _lockTimeMicros(
+        _metrics.counter("arangodb_collection_lock_acquisition_micros", 0,
+                         "Total amount of collection lock acquisition time [μs]")),
       _lockTimes(
-          _metrics.histogram("arangodb_collection_lock_acquisition_time",
-                             log_scale_t(10., 0.0, 1000.0, 11),
-                             "Collection lock acquisition time histogram [s]")),
+        _metrics.histogram("arangodb_collection_lock_acquisition_time",
+                           log_scale_t(10., 0.0, 1000.0, 11),
+                           "Collection lock acquisition time histogram [s]")),
       _sequentialLocks(
-          _metrics.counter("arangodb_collection_lock_sequential_mode", 0,
-                           "Number of transactions using sequential locking of "
-                           "collections to avoid deadlocking")) {}
+        _metrics.counter("arangodb_collection_lock_sequential_mode", 0,
+                         "Number of transactions using sequential locking of "
+                         "collections to avoid deadlocking")),
+      _numRead(
+        _metrics.counter("arangodb_num_document_reads", 0,
+                         "Number of document reads since process start")),
+      _numWrite(
+        _metrics.counter("arangodb_num_document_writes", 0,
+                         "Number of document writes since process start")),
+      _numReplicate(
+        _metrics.counter("arangodb_num_document_replications", 0,
+                         "Number of document replications since process start")),
```

Comment:
I would prefer "replication write operations" here.

Reply:
If you don't mind, let's leave this concise. It is not ambiguous.

Reply:
done